tabular/nyc_tlc/model_urls.txt

```diff
@@ -0,0 +1,4 @@
+https://www.dropbox.com/s/iuwmlaxcl3xghwg/model.catb?dl=1,model_extras/model.catb
+https://www.dropbox.com/s/jebkaxgoqsa4j15/model_task.pkl?dl=1,model_extras/model_task.pkl
+https://www.dropbox.com/s/t0wursua1lgwnb2/cont_features.pkl?dl=1,model_extras/cont_features.pkl
+https://www.dropbox.com/s/ic3yus8uux12lca/cat_features.pkl?dl=1,model_extras/cat_features.pkl
```

Comment:
Are we supposed to download the `model.py` file from dropbox in this proposal?

Reply:
i was thinking of keeping the model.py file checked in here since it's a code file

Reply:
but actually i realized - if it's easier to keep the model.py files in Dropbox for purposes of easily fetching in the notebook, that's fine. I'll probably do a pass after notebooks are checked in to try to use ri-public-examples as opposed to relying purely on Dropbox